src/wasm.rs

```diff
@@ -71,7 +71,9 @@ struct CodeData {
     code_base_id: usize,
 }
 
-/// Interface to call into a `Wasm` module.
+/// Acts as the interface for interacting with WebAssembly (Wasm) modules. 
```

Comment:
Trailing whitespace at the end of this line is spotted as a warning by `cargo fmt -- --check` command.